### DISCUSSION TITLE
Would love the addition of a test URL for paid accounts

### DISCUSSION THREAD

**Discussion by ctiwana** [umbraco/Umbraco.Workflow.Issues#50] [Ideas]
It would be great to have the ability to add a test URL (not localhost or a subdomain) to our license. A big bonus is being able to update it on our own.

**Comment by nathanwoulfe** [comment]
Hi @ctiwana we can absolutely add test domains to your license. Please reach out to your partner manager (or email me njw at umbraco dot dk) and we can get that sorted.

License management is in the roadmap for all our commercial offerings. I can't provide timelines, but it is coming.
